one.css

```diff
@@ -225,6 +225,9 @@ Buttons
     border-radius: 50%;
     background-color: var(--orange);
     font-size: 1rem;
+    width: 2.5rem;
+    height: 2.5rem;
+    outline: none;
```

Comment:
Hey! Apparently outline: none should not be used. Maybe try outline-color: transparent? Hope this helps!

Reply:
done